app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) < 3:
+        raise ValueError("Invalid command format.")
+
+    source_file = parts[1]
+    destination = parts[2]
```

Comment:
Perform unpacking and check that the command name is `mv` and the source filename does not equal the destination filename before moving forward. Don't use indexes to retrieve values here.